src/pages/conference/timeline/_data-schedule.tsx

```diff
@@ -0,0 +1,356 @@
+import React from 'react'
+import { Localize } from 'components/localization'
+
+const events_data = [
+    {
+        id: 'day_01',
+        title: <Localize translate_text="Day1" />,
+        date: <Localize translate_text="November" />,
+        period: <Localize translate_text="Registrations 9:30 - 10:00" />,
+        data: [
+            {
+                timings: [
+                    {
+                        time: <Localize translate_text="10:00 - 10:20" />,
```

Comment:
Are we need to localize a time?

Reply:
Let's keep this for now, we don't know the format of time on other languages